ports/mimxrt/boards/MIMXRT1020_EVK/MIMXRT1020_EVK_af.csv

```diff
@@ -0,0 +1,23 @@
+Pad,ALT0, ALT1, ALT2, ALT3, ALT4, ALT5, ALT6, ALT7, ALT8, ALT9,ADC,ACMP,Default
+GPIO_AD_B1_09,LPI2C2_SDA,FLEXPWM1_PWMB1,LPUART2_RXD,SAI1_TX_DATA3,FLEXIO1_D06,GPIO1_IO25,LPSPI3_PCS1,XBAR_INOUT13,,,ADC1_IN9,ACMP2_IN5,ALT5
```

Comment:
as above, this can be moved up a directory

Reply:
see above